torchci/pages/[repoOwner]/[repoName]/pull/revert/[prNumber].tsx

```diff
@@ -0,0 +1,147 @@
+import { revertClassifications } from "lib/bot/Constants";
+import { fetcher, getMessage, getFailureMessage } from "lib/GeneralUtils";
+import { isFailure } from "lib/JobClassifierUtil";
+import { CommitData, JobData } from "lib/types";
+import Head from "next/head";
+import { useRouter } from "next/router";
+import { useState } from "react";
+import { Col, Container, Row } from "react-bootstrap";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import ReactMarkdown from "react-markdown";
+import useSWR from "swr";
+import { commentOnPR } from "lib/githubFunctions";
+import { useSession } from "next-auth/react";
+
+export default function Revert() {
```

Comment:
Thoughts on making this a dialog box instead of a separate page?

Reply:
Yea I think I'll add that as well. I was originally thinking this page was gonna be reached through the alerts but I guess it can be both a modal and a separate page.